app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -87,24 +91,35 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
   @Override
   public View getDropDownView(int position, View convertView, ViewGroup parent) {
     CheckBoxHolder holder;
-    if (convertView == null) {
-      convertView = layoutInflater.inflate(
+    convertView = layoutInflater.inflate(
```

Comment:
keep wrapped in null check, otherwise, we make a new one every time this is called